linkerd/app/outbound/Cargo.toml

```diff
@@ -15,6 +15,8 @@ allow-loopback = []
 test-subscriber = []
 test-util = ["linkerd-app-test", "linkerd-meshtls-rustls/test-util"]
 
+prometheus-client-rust-242 = [] # TODO
```

Comment:
Is this needed here?

Reply:
Yeah--we are using this marker feature to disable some test assertions. In newer rust versions, we have to declare all of the features in the cargo.toml...